R/get_taxonlijsten_items.R

```diff
@@ -0,0 +1,152 @@
+#' @title Query to extract the taxa on a taxonlist from D0156_00_Taxonlijsten
+#'
+#' @description This function queries D0156_00_Taxonlijsten and gives an
+#' overview of the taxa that are on a given taxon list version. The interpreted
+#' taxa are given by default, but it is possible to add taxa as they were
+#' originally published. The taxa of the latest list version are shown
+#' unless specified otherwise.
+#'
+#' @param connection dbconnection with the database 'D0156_00_Taxonlijsten'
+#' on the inbo-sql07-prd server
+#' @param list name of the taxonlist you want to retrieve. Wildcards %
+#' are allowed. Case insensitive.
+#' @param taxon name of the taxon you want to retrieve. Scientific and
+#' vernacular (Dutch) names are allowed. Wildcards % are allowed.
+#' Case insensitive.
```

Comment:
```suggestion
#' @inheritParams get_taxonlijsten_features
```
Another suggestion: instead of repeating the same descriptions for variable names in different functions, one can just refer like this to the other function in which these variables are described.  If changes are needed, it only has to be done once.  (`version` and `collect` below can also be omitted, as they are also described in `get_taxonlijsten_features`.)